services/gateway/apis/http-version3/swagger/definitions/indexStatus.json

```
@@ -0,0 +1,67 @@
+{
+    "IndexStatus": {
+        "type": "object",
+        "required": [
+            "data",
+            "meta"
+        ],
+        "properties": {
+            "data": {
+                "type": "object",
+                "description": "Number of items returned in the request.",
+                "properties": {
+                    "genesisHeight": {
+                        "type": "number",
+                        "example": "0",
+                        "description": "Height of the genesis block."
+                    },
+                    "lastBlockHeight": {
+                        "type": "number",
+                        "example": 2330,
+                        "description": "Latest block height."
+                    },
+                    "lastIndexedBlockHeight": {
+                        "type": "number",
+                        "example": 2330,
+                        "description": "Latest block height indexed."
+                    },
+                    "chainLength": {
+                        "type": "number",
+                        "example": 2331,
+                        "description": "Number of blocks in the chain."
+                    },
+                    "numBlocksIndexed": {
+                        "type": "number",
+                        "example": 2330,
+                        "minimum": 0,
+                        "description": "Current block height."
+                    },
```

I think the description is not relevant to the property name
```suggestion
                    "numBlocksIndexed": {
                        "type": "number",
                        "example": 2330,
                        "minimum": 0,
                        "description": "Current block height."
                    },
```